genet/predict/PredUtils.py

```diff
@@ -27,23 +27,55 @@ def preprocess_masked_seq(data, seq_length):
 
 def one_hot_encode(seq):
     mapping = {"A": 0, "C": 1, "G": 2, "T": 3, "X": 4}
-    map_seq = [mapping[i] for i in seq]
+    map_seq = np.array([mapping[i] for i in seq])
     arr_seq = np.eye(5)[map_seq]
     return np.delete(arr_seq, -1, axis=1)
 
 def preprocess_seq(data, length:int):
-    encoded_seq = [one_hot_encode(seq.upper()) for seq in data]
+    encoded_seq = np.array([one_hot_encode(seq.upper()) for seq in data])
     return np.stack(encoded_seq, axis=0).reshape(len(data), 1, length, 4)
 
 def reverse_complement(sSeq):
     """Biopython의 reverse_complement 또는 reverse_complement_rna로 모두 대체함. 
     더 이상 쓰이지 않는 함수.
     테스트 해보고 문제 없으면 없앨 예정.
-    """    
-    dict_sBases = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A', 'N': 'N', 'U': 'U', 'n': '',
-                   '.': '.', '*': '*', 'a': 't', 'c': 'g', 'g': 'c', 't': 'a'}
-    list_sSeq = list(sSeq)  # Turns the sequence in to a gigantic list
-    list_sSeq = [dict_sBases[sBase] for sBase in list_sSeq]
-    return ''.join(list_sSeq)[::-1]
+    """ 
+    dict_sBases = {k: v for k, v in zip('ACGTNU.acgt', 'TGCANU.tgca')}
+    return sSeq.translate(str.maketrans(dict_sBases))[::-1]
```

Comment:
Great and simple than before! 

But as I comment in doc string (sorry for Korean haha..), I decided to remove this function and will be replaced to Bio.Seq.reverse_complement & reverse_complement_rna (from biopython package). 

Please see PrimeEditor.py line 22 
(from Bio.Seq import Seq, transcribe, back_transcribe, reverse_complement, reverse_complement_rna)